includes/class-wc-payments-features.php

```diff
@@ -111,6 +111,30 @@ public static function is_upe_settings_preview_enabled() {
 		return '1' === get_option( '_wcpay_feature_upe_settings_preview', '1' );
 	}
 
+	/**
+	 * Indicates whether card payments are enabled for this (Stripe) account.
+	 *
+	 * @return bool True if account can accept card payments, false otherwise.
+	 */
+	public static function are_payments_enabled() {
+		$account = WC_Payments::get_database_cache()->get( WCPay\Database_Cache::ACCOUNT_KEY, true );
+
+		return is_array( $account ) && ( $account['payments_enabled'] ?? false );
+	}
+
+	/**
+	 * Checks if WooPay is enabled.
+	 *
+	 * @return bool
+	 */
+	public static function is_woopay_enabled() {
+		$is_platform_checkout_eligible               = self::is_platform_checkout_eligible(); // Feature flag.
+		$is_platform_checkout_enabled                = 'yes' === WC_Payments::get_gateway()->get_option( 'platform_checkout', 'no' );
```

Comment:
```suggestion
		$is_platform_checkout_enabled                = 'yes' === WC_Payments::get_gateway()->get_option( 'platform_checkout' );
```
I don't think you need to pass the second parameter [here](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payment-gateway-wcpay.php#L1675), as it will be `null` by default.

Reply:
Yes, good point! I think this default is being set throughout the codebase from it's initial incarnation, showing 'no' in the Gateway Settings or something similar. I can't find any place where we're actually checking for that string. Definitely not needed here. Fixed in [619bcca](https://github.com/Automattic/woocommerce-payments/pull/6187/commits/619bccad13ff41ba492fbd5bf444d4c6c2283017).